Sources/TuistKit/Services/TestService.swift

```diff
@@ -87,9 +87,14 @@ final class TestService {
         clean: Bool,
         configuration: String?,
         path: AbsolutePath,
+        automationPath: AbsolutePath?,
         deviceName: String?,
         osVersion: String?
     ) throws {
+        if let automationPath = automationPath {
```

Comment:
yes, the suggestion sounds reasonable to me 👍

Reply:
> I'd create a temporary directory

temporary as in it should be deleted after `tuist test` is finished? If so, this will not fix the issue of not being able to debug the test project. Instead of creating `automationPath`, we could introduce an env variable `TUIST_AUTOMATION_PATH` which would be needed for acceptance tests - and also with possible debugging in the future.

So, basically, I see two options:
- we generate test project to Tuist's global directory but without it being temporary
- we generate test project the same way we do now and allow configuration via env variable

If we generated test project to Tuist's global directory and deleted after command ended, it would have no advantage to the way we do it now (macOS temp directory)

Issue with generating project to Tuist's global directory without it being temporary is the fact that the directory might bloat in size quite quickly. Therefore, I'd be for hiding integrating automation path and using it only for acceptance tests / *our* debugging

What do you think, @pepibumur?

Reply:
Here's what I'd suggest:
- Generate it in a temporary directory that gets deleted afterward.
- If you want to debug the generated project, you can do `tuist test --debug`, and that'll generate the project in a temporary directory and open it in Xcode for debugging. Once the Tuist process finishes, the project gets deleted.
- If we need to generate the project in a specific directory for the purpose of writing acceptance tests, then we can control that through an environment variable.